how_to_use_package.md

```diff
@@ -0,0 +1,47 @@
+# 🚀 Quick Start Guide for matplotlib_visual
+--- 
+
+## **🔧Installations:**
+```bash
+pip install package
+```
+> **Note:** The `matplotlib-visual` package has been implemented by [Project Manager](https://github.com/this-is-yaash) and has details specific to him, you are sugested to follow the steps above and fork the repository and upload a package.
+
+## **⏱️Execution:**
+1. Create a Python file with **.py** extension and enter any of the code snippets below.
+2. Or, open a terminal and run Python interactively:
+```bash
+python
+```
+
+### 📊Features of this package:
+- **📍Displaying source code** in the console output.
+```python
+    from package.models import display
+    display("<filename-without-extension>")
+```
+ *`models` method displays source code in the console.*  
+
+
+- **📍Stealth Copy to Clipboard** for a more secure method.
```

Comment:
*Sthealthly
Again, explain the method first and then add command. This helps the user in understanding what the command is about instead of trying it out firsthand.

Reply:
The correct phrasing would be "Stealth Copy to Clipboard" rather than "Stealthly Copies to Clipboard."

Your bullet point could be revised as follows:

📍Stealth Copy to Clipboard for a more secure method.
This is because "stealth" is used as an adjective here, and the term "stealth copy" conveys the idea of copying discreetly or covertly, which aligns with the intended functionality. The rest of your description looks good and accurately describes how to use the display function to copy the source code to the clipboard.